client/src/reducers/transforms/sortColumn.js

```diff
@@ -32,46 +31,50 @@ function composeComparators(comparators) {
 
 const nextSortLevel = columns => {
   const sortLevels = columns
-    .map(column => column.sort)
+    .map(column => column.get('sort'))
     .filter(n => n != null);
-  if (sortLevels.length === 0) {
+  if (sortLevels.size === 0) {
     return 1;
   }
-  return sortLevels[sortLevels.length - 1] + 1;
+  return sortLevels.get(-1) + 1;
```

Comment:
yeah, this might be a bit too clever :) Negative indexes are allowed and counting from the end of the list. So index -1 is the last element in the vector.